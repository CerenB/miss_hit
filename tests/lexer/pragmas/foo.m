%% (c) Copyright 2020 Florian Schanda

function v = foo(x)
    % mh:flow:unused x, "we dont' need x here, thanks"
    v = 12;
end
