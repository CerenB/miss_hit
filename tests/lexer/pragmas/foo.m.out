In foo.m, line 1
| %% (c) Copyright 2020 Florian Schanda
| ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ info: [SL] COMMENT '% (c) Copyright 2020 Florian Schanda'
In foo.m, line 1
| %% (c) Copyright 2020 Florian Schanda
|                                      ^ info: NEWLINE
In foo.m, line 3
| function v = foo(x)
| ^^^^^^^^ info: [SL] KEYWORD
In foo.m, line 3
| function v = foo(x)
|          ^ info: IDENTIFIER
In foo.m, line 3
| function v = foo(x)
|            ^ info: ASSIGNMENT
In foo.m, line 3
| function v = foo(x)
|              ^^^ info: IDENTIFIER
In foo.m, line 3
| function v = foo(x)
|                 ^ info: BRA
In foo.m, line 3
| function v = foo(x)
|                  ^ info: IDENTIFIER
In foo.m, line 3
| function v = foo(x)
|                   ^ info: KET
In foo.m, line 3
| function v = foo(x)
|                    ^ info: NEWLINE
In foo.m, line 4
|     % mh:flow:unused x, "we dont' need x here, thanks"
|     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ info: [SL] PRAGMA
In foo.m, line 4
|     % mh:flow:unused x, "we dont' need x here, thanks"
|                                                       ^ info: NEWLINE
In foo.m, line 5
|     v = 12;
|     ^ info: [SL] IDENTIFIER
In foo.m, line 5
|     v = 12;
|       ^ info: ASSIGNMENT
In foo.m, line 5
|     v = 12;
|         ^^ info: NUMBER
In foo.m, line 5
|     v = 12;
|           ^ info: SEMICOLON
In foo.m, line 5
|     v = 12;
|            ^ info: [S ] NEWLINE
In foo.m, line 6
| end
| ^^^ info: [SL] KEYWORD
In foo.m, line 6
| end
|    ^ info: NEWLINE
foo.m: lexed OK
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
